agency/src/services/aries-agent/aries-agent.class.ts

```diff
@@ -645,4 +724,60 @@ export class AriesAgent {
       );
     }
   }
+
+  // TODO: Need to type response
```

Comment:
Do we want to type the response before merging, or leave it for later (e.g.: new issue)?